pkg/util/azureclient/openshiftmanagedcluster/admin/client.go

```diff
@@ -1175,3 +1175,32 @@ func (client OpenShiftManagedClustersClient) RunCommand(ctx context.Context, res
 	}
 	return nil
 }
+
+func (client OpenShiftManagedClustersClient) GetPluginVersion(ctx context.Context, resourceGroupName, resourceName string) (*api.GenevaActionPluginVersion, error) {
```

Comment:
this implementation works but deviates from the pattern employed by `azure-sdk-for-go`